apps/choleskey/generateInputMatrix.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This is an Jupyter Script for generating the Input Positive Definition(PD) Matrix for Cholesky decomposition.
+# The generated .txt file stores the entire PD matrix with the matrix elems row by row. 
+# This input.txt file can be used directly In benchmark part in each cpp file.
+
+
+import numpy as np
+
+#enter matrix size
+matrixSize = 4
+
+#1. Generates a random matrix A of size matrixSize x matrixSize.
+A = np.random.rand(matrixSize, matrixSize)
+print(A)
+
+#2. Computes the product of matrix A and its transpose, i.e., A * A^T, storing the result in matrix B.
+B = np.dot(A, A.transpose())
+print("B")
+print(B)
+
+#3. Adds the transpose of matrix B to itself, generating a symmetric matrix C.
+C = B + B.T  # Ensure it's symmetric
+
+print("C")
+print(C)
+
+try:
```

Comment:
It looks like the script will either generate or not generate a positive definitive matrix when run so that should be in a loop until we do get a correct solution. 

Also using Cholesky transformation itself to check if the generated matrix is positive definitive doesn't seem to be the correct solution especially for larger matrices. I would recommend using a cheaper (eigen value based or Gershgorin circle theorem to verify this). You can refer to Gilbert Strang's book/videos or online resources for this.